ipi_tests/regression_tests/geop/bfgs/input.xml

```diff
@@ -1,10 +1,10 @@
 <simulation mode="static" verbosity="medium">
-    <output prefix='min'>
+    <output prefix='simulation'>
         <properties stride='1' filename='out'>  [ step, potential{electronvolt}] </properties>
         <trajectory stride="1" filename="xc" format="xyz">x_centroid{angstrom}</trajectory>
    </output>
    <total_steps>400       </total_steps>
-   <ffsocket name="cbe" mode="inet" >
+   <ffsocket name="cbe" mode="unix" >
        <port> 33334 </port> 
        <address> localhost </address> 
```

Comment:
This is rather confusing. Perhaps should use another address for unix socket.

Reply:
I am sorry, I don't get what is confusing. Could you be more specific?